src/EA.Weee.Web/Areas/Admin/ViewModels/Reports/ChooseReportViewModel.cs

```diff
@@ -15,7 +15,8 @@ public ChooseReportViewModel()
                 Reports.UkWeeeData,
                 Reports.ProducerPublicRegister,
                 Reports.SchemeObligationData,
-                Reports.MissingProducerData
+                Reports.MissingProducerData,
+                Reports.AatfAeReturnData
```

Comment:
Can we create a unit test to ensure this view model is setup with correct list

Reply:
Added